.github/workflows/lint-server.yml

```diff
@@ -8,10 +8,22 @@ on:
   workflow_dispatch:
 
 env:
- BRANCH_NAME: ${{ github.head_ref || github.ref_name }} 
+ BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
  EXCLUDE_ENTERPRISE: true
- 
+
 jobs:
+  down-migrations:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: "focalboard"
+      - name: assert that down migrations are SELECT 1 scripts
+        run: |
+          cd focalboard
+          echo 'SELECT 1;' > downmigration
```

Comment:
@mgdelacroix should the `downmigration` file be `rm`-ed afterwards? or is the filesystem not persisted between jobs? my concern would just be keeping the working directory clean

Reply:
Each job is fully isolated AFAIK, which would be consistent with each job doing its own `checkout`